src/Umbraco.Web/Properties/AssemblyInfo.cs

```diff
@@ -34,5 +34,6 @@
 [assembly: InternalsVisibleTo("Concorde.Sync")]
 [assembly: InternalsVisibleTo("Umbraco.Courier.Core")]
 [assembly: InternalsVisibleTo("Umbraco.Courier.Persistence")]
+[assembly: InternalsVisibleTo("Umbraco.VisualStudio")]

```

Comment:
There's a dependency on an internal class from Umbraco.Web.UI in PetaPocoRepositoryBase.